manifests/infrastructure-roles-new.yaml

```diff
@@ -3,8 +3,8 @@ data:
   # infrastructure role definition in the new format
   # robot_zmon_acid_monitoring_new
   user: cm9ib3Rfem1vbl9hY2lkX21vbml0b3JpbmdfbmV3
-  # robot_zmon_new
-  role: cm9ib3Rfem1vbl9uZXc=
+  # robot_zmon
```

Comment:
To test default values it makes sense to omit this key.

Reply:
yes, thought about it too. Will update